Doc/whatsnew/3.13.rst

```diff
@@ -334,6 +334,12 @@ Removed
   attribute instead.
   (Contributed by Nikita Sobolev in :gh:`105546`.)
 
+* Remove deprecated ``re.template`` function
+  and ``re.T`` and ``re.TEMPLATE`` flags.
+  Also removes ``SRE_FLAG_TEMPLATE`` constant
+  from ``sre_constants`` and ``re._constants``.
```

Comment:
Please explain how to update existing code. I suppose that replacing re.template() with re.compile() works in most cases, no?

Please mention also that the flag never worked, see the previous removal commit.

Reply:
I think that these who use it should find how to replace it themselves. It cannot just be replaced with `re.compile()`, otherwise they would use `re.compile()` at first place. Perhaps.